xamarin/Xamarin.Forms
# [Bug] NullReferenceException Closing Window During Animation

### Description of Change ###

Added null check to the WPF current application object to prevent crashes on it

### Issues Resolved ### 


- fixes #7061

### API Changes ###

 
 None

### Platforms Affected ### 


- WPF

### Behavioral/Visual Changes ###


None

### Before/After Screenshots ### 


Not applicable

### Testing Procedure ###


Open the gallery app, navigate to issue 7061 and exit the application. It should not throw an exception.

### PR Checklist ###
